Consolidated view (#646)
plausible/docs: docs/consolidated-views.md

---
title: Consolidated views
---

import useBaseUrl from '@docusaurus/useBaseUrl';

A consolidated view gives you an overview of traffic across all your websites. It looks just like any other Plausible stats dashboard, but the underlying data includes all your sites, instead of just one.

## How to enable your consolidated view?

In order to have a consolidated view, the following requirements need to be met:

1) You have a [team](/users-roles#creating-a-new-team) set up

> 2) Your team has at least 2 sites

Reviewer: ```suggestion
2) At least two sites in that team
```

3) A Plausible Business plan (or a custom plan with Business features)

> Once your team meets these requirements, your consolidated view will appear automatically on the /sites page.

Reviewer: ```suggestion
Once your team meets these requirements, your consolidated view will appear automatically on the /sites page (where all your dashboards are listed). You will be able to open the consolidated view dashboard from here.
```

> Once your team is eligible for having a consolidated view, you can navigate to the /sites page and it will show up automatically.

Reviewer: ```suggestion
Once your team meets these requirements, your consolidated view will appear automatically on the /sites page.
```

Reviewer 2: Is "/sites page" the appropriate jargon here?

## Configuration

Just like a regular site, a consolidated view has its own settings. You can access the settings directly from the /sites page, or the site picker menu on the top left of the consolidated dashboard.

#### Timezone

You can change the timezone in which your consolidated stats are reported. By default, it's set to the timezone in which you've configured the majority of your sites.

#### Goals

In order to see conversions showing up on your consolidated dashboard, you'll have to add the goals manually, just as you would on a regular Plausible site. 

#### Custom properties

Just like goals, custom properties need to be added manually too. 

#### Email reports

> You can also enable email reports that take all your sites' traffic into account.

Reviewer: ```suggestion
You can enable [email reports](https://plausible.io/docs/email-reports) that combine traffic from all your sites.
```
